doc/netplan.md

```diff
@@ -437,6 +437,24 @@ similar to ``gateway*``, and ``search:`` is a list of search domains.
             dhcp6: true
             optional-addresses: [ ipv4-ll, dhcp6 ]
 
+``activation-mode`` (scalar) – since **0.102**
```

Comment:
Depending on when we merge this PR, we might need to remember to update the version string.